Draft-Accepted/RFCNNNN-Install-PowerShell.md

```diff
@@ -0,0 +1,205 @@
+on a system that currently does not include Powershell is an added benefit for using a command.
+
+## Specification
+
+Command name: Install-PowerShell7
+
+The default behavior is to download and extract the .ZIP package, then install the latest LTS
+release of PowerShell into the user scope allowed by permissions. If the user has administrative
+privilege, they can choose to install to the scope of AllUsers.
+
+- The package install location - $env:LOCALAPPDATA\Microsoft\pwsh
+  - Downloaded packages should be certificate signature verified before installation. 
+  - Failure of the signature should return an error message and stop performing installation.
+- The installer should update the Users PATH for pwsh
+- The installer should enable Microsoft Updates for scope AllUsers. Due to permission requirements,
+  the scope CurrentUser will require manual updating.
+
+The following parameters can be added by experienced PowerShell users to customize the installation.
+
+- Parameter: Scope - overrides the default of CurrentUser and allows the user to specify the
+  installation location AllUsers.
+- Parameter: Channel - options Stable/Preview/LTS - allows the user to select if they want the
+  current preview or LTS release. 
+
+## Syntax and Parameter Sets
+
+The syntax of the command is intentionally made similar to that of a cmdlet. Powershell users are
+familiar with a cmdlet syntax and will find this command similar in experience.
+
+### ParameterSet Channel (Default)
+
+```
+Install-PowerShell7 -Channel {Preview | LTS | Stable} [-Scope {AllUsers | CurrentUser}] [-TrustMicrosoftPublisher]
+```
+
+The **Channel** parameter has the following validation attribute:
+[ValidateSet("Preview", "LTS", "Stable")]
+
+The **Scope** parameter has the following validation attribute:
+[ValidateSet("AllUsers", "CurrentUser")]
+
+Scope specifies the installation scope of the module. The default installation scope is CurrentUser,
+which does not require elevation. Users may choose to select the AllUsers scope, which requires
+elevation.
+
+Scope selection has an impact on receiving auto-updates from Microsoft Update:
+
+- AllUsers for x86/x64 will auto-update via Microsoft Update.
+  - Note, on arm64 there is no AllUsers
+- CurrentUser is manually updated
+
+The **TrustMicrosoftPublisher** is required for installation on Windows. Powershell packages are
+signed with a Microsoft certificate, not the Windows certificate. When enabled, this parameter
+should enable the publishers content for this session.
+
+### Data model
+
+The `Install-PowerShell7` command provide useful information post installation. The data model for
+the output is below.
+
+```output
+PSTypename   = "PSInstallationInfo"
+[Version]Version      = $version
+[string]Path         = (Get-Command pwsh).source
+[string]Channel      = if ($Channel) { $channel} else {"LTS" }
+[string]Computername = $env:COMPUTERNAME
+```
```

Comment:
```suggestion
```
@theJasonHelmick - The change to native command means we no longer need the data model.  Command should not return anything - should set exit code.